tools/cli/package.json

```
@@ -0,0 +1,35 @@
+{
+	"name": "jetpack-cli",
+	"version": "1.0.0",
+	"private": true,
+	"description": "[Jetpack](https://jetpack.com/) is a WordPress plugin that supercharges your self-hosted WordPress site with the awesome cloud power of [WordPress.com](https://wordpress.com).",
+	"homepage": "https://jetpack.com",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"bin": {
+		"jetpack": "bin/jetpack"
+	},
+	"scripts": {
+		"test-cli": "mocha tests/runner.js --require esm",
```

There is no need to specify what tests this command would run since it is in a cli specific package already
```suggestion
		"test": "mocha tests/runner.js --require esm",
```

---

2842da0

Updated it here and in the GH action that runs the tests for CI.